Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.13-alpine3.21
+LABEL maintainer="dmysamo@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Mounting the entire current directory to `/app` can introduce unwanted files into the container. It's better to be selective with what you mount to avoid potential conflicts or security issues.